packages/portfolio-contract/src/type-guards.ts

```diff
@@ -1,7 +1,9 @@
 import type { TypedPattern } from '@agoric/internal';
 import { M } from '@endo/patterns';
+import type { SupportedEVMChains } from '@agoric/orchestration/src/axelar-types.js';
 import type { YieldProtocol as YieldProtocolT } from './constants.js';
 import { YieldProtocol } from './constants.js';
+import { supportedEVMChains } from './portfolio.exo.js';
```

Comment:
I think this import should go the other way around. the evm chains should go here and the exo should import them